runtime/convertTypes.go

```diff
@@ -191,9 +191,9 @@ func exportCompositeType(t *sema.CompositeType, results map[sema.TypeID]cadence.
 	fieldMembers := make([]*sema.Member, 0, len(t.Fields))
 
 	for _, identifier := range t.Fields {
-		member := t.Members[identifier]
+		member, ok := t.Members.Get(identifier)
 
-		if member.IgnoreInSerialization {
+		if !ok || member.IgnoreInSerialization {
```

Comment:
Is there ever a case where the lookup would fail?

Reply:
AFAIU `Fields` are a subset of `Members`. So the lookup should always succeed. 

Maybe a panic would be a good idea.